gcd/src/test/resources/datastore-indexes.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Copyright 2017, TeamDev Ltd. All rights reserved.
```

Comment:
This is obviously a default format for the copyright notes inserted by the IDE into XML files.